### PR TITLE
Allow thread size to be configurable for AwsSqsJobQueue

### PR DESCRIPTION
**Pull request by monkey-mas**
It'd be nice to have a config for the number of threads instead of hardcoding it.

Feedback: https://github.com/square/misk/pull/966#discussion_r280936506
Issue ref: https://github.com/square/misk/issues/284

### CODE REVIEW

File: misk-aws/src/main/kotlin/misk/jobqueue/sqs/AwsSqsJobQueueConfig.kt [hunk 1]
```diff
@@ -17,6 +17,12 @@ class AwsSqsJobQueueConfig(
...
    * an internal queue name to the (account ID, region, name) of the queue in the external account
    */
-  val external_queues: Map<String, AwsSqsQueueConfig> = mapOf()
+  val external_queues: Map<String, AwsSqsQueueConfig> = mapOf(),
+
+  /**
+   * Number of consumer threads to run. This parameter controls the number of jobs handled by
+   * consumer concurrently.
+   */
+  val consumer_thread_pool_size: Int = 4
```
Comment (monkey-mas) [inline misk-aws/src/main/kotlin/misk/jobqueue/sqs/AwsSqsJobQueueConfig.kt]: Any suggestions for the comment and var name? :)

Comment (wesleyk) [reply misk-aws/src/main/kotlin/misk/jobqueue/sqs/AwsSqsJobQueueConfig.kt]: the naming seems reasonable!

The second part of the comment doesn't seem quite accurate: "This parameter controls the number of jobs handled by consumer concurrently." Maybe like, "Number of threads to run for the job queue consumer"?

Comment (ryanhall07) [reply misk-aws/src/main/kotlin/misk/jobqueue/sqs/AwsSqsJobQueueConfig.kt]: number of jobs that can be processed concurrently

Comment (monkey-mas) [reply misk-aws/src/main/kotlin/misk/jobqueue/sqs/AwsSqsJobQueueConfig.kt]: Thanks for the feedback! I'll pick @ryanhall07 's idea `Number of jobs that can be processed concurrently.` if it's okay! :)

**Comment by monkey-mas** [comment]
Thanks for the review! :)

**Pull request merged**